tldw_Server_API/tests/Web_Scraping/test_enhanced_web_scraping_guards.py

```
@@ -0,0 +1,116 @@
+        cookies={},
+        impersonate=None,
+        proxies=None,
+        strategy_order=None,
+        schema_rules=None,
+        llm_settings=None,
+        regex_settings=None,
+        cluster_settings=None,
+        backend="auto",
+    )
+
+    monkeypatch.setattr(scraper, "_resolve_scrape_plan", lambda url: (plan, "httpx", ""))
```

Ruff (0.15.9)

[warning] Unused lambda argument: `url`

(ARG005)